obspy/core/tests/test_code_formatting.py

```diff
@@ -9,13 +9,19 @@
 import re
 import unittest
 
+import obspy
 from obspy.core.util.testing import check_flake8, get_all_py_files
 
+_pattern = re.compile(r"^\d+\.\d+\.\d+$")
+CLEAN_VERSION_NUMBER = bool(_pattern.match(obspy.__version__))
```

Comment:
I just tried that but `core/testing.py` is imported before it has a version number (all our modules import `add_doctests`/`add_unittests` ...). It would thus require significant changes to our initial startup phase or awkward logic. I'm going to leave it where it is right now.